packages/block-library/src/image/view.js

```diff
@@ -23,6 +23,7 @@ const { state, actions, callbacks } = store(
 	'core/image',
 	{
 		state: {
+			metadata: {},
```

Comment:
It's not necessary to initialize `metadata` on the client since it's being initialized on the server.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/63348/commits/56d94872c41e57fcbb47652359d3acdedf1bbe63.